packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -68,6 +69,12 @@ function ShoppingPane(props) {
   const [loaded, setLoaded] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
   const [searchTerm, setSearchTerm] = useState('');
+  const [orderby, setOrderby] = useState('date');
+  const [order, setOrder] = useState('desc');
+  const onSortBy = (option) => {
+    setOrderby(option?.orderby);
+    setOrder(option?.order);
```

Comment:
The use of optional chaining here indicates that `option` can be `null`. Is this ever the case?

Reply:
I'll remove --- the only case would be an input error in the source code.